packages/yargs/lib/logger.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/* eslint-disable no-console */
+
+const { basename } = require('path');
+
+const PrettyError = require('pretty-error');
+
+const logLevels = { error: 0, warn: 1, info: 2, request: 3 };
+
+module.exports = exports = class Logger {
+  constructor(name, workspace) {
+    this.name = name;
+    this.level = logLevels.info;
+    this.prettyError = new PrettyError();
+    this.prettyError.withoutColors();
```

Comment:
This makes me sad. I'd like to see a `--no-colors`-flag here and maybe also a check on `process.env.CI === 'true'`.

Reply:
I would love to add console output coloring in a later PR and keep things monochromatic for now. #iterateallthethings